### PR TITLE
feat(config): Configure required group memberships

### PR DESCRIPTION
**Pull request by lwander**
Please check my wording for the help text.

### CODE REVIEW

**Comment by lwander** [comment]
ping

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.halyard.cli.command.v1.config.providers.account;
+
+public class AccountCommandProperties {
+  public static final String REQUIRED_GROUP_MEMBERSHIP_DESCRIPTION = "A list of groups a user must be in at least on of to make changes to your cloud provider "
```
Comment (ttomsu) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java]: "A user must be a member of at least one specified group in order to make changes to your cloud resources."

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java]: Thanks :+1:

Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java]: What do you think of making the end read "to this account's cloud resources." ?

Comment (ttomsu) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/account/AccountCommandProperties.java]: SGTM

**Pull request merged**